tests/unit/type/zset.tcl

```diff
@@ -2480,4 +2480,27 @@ start_server {tags {"zset"}} {
         r zscore zz dblmax
     } {1.7976931348623157e+308}
 
+    test {zunionInterDiffGenericCommand acts on SET and ZSET} {
+        # Test one key is big and one key is small separately.
+        foreach {big_key small_key zset_entries set_entries} {
+             zset{t} set{t} {1 a 2 b 3 c 4 d} {a b c}
+             set{t} zset{t} {1 a 2 b 3 c} {a b c d}
+        } {
```

Comment:
zset{t} set{t} {1 a 2 b 3 c 4 d} {a b c}, this case cause zdiff, zdiffstore crash

set{t} zset{t} {1 a 2 b 3 c} {a b c d}, this case cause zinter, zinterstore, zintercard, zdiff, zdiffstore crash

Reply:
look like this comment somehow will break the {}